guides/common/modules/con_virtual-machine-subscriptions-overview.adoc

```diff
@@ -0,0 +1,25 @@
+[id="virtual-machine-subscriptions-overview_{context}"]
+= Virtual machine subscriptions overview
+
+Virtual machines (VMs) require host-based subscriptions instead of physical subscriptions.
+Many host-based subscriptions include entitlements for unlimited VMs.
+
+You can configure and deploy virt-who on your hypervisors.
+virt-who queries the virtualization platform and reports hypervisor and VM information to {Project}.
+You can view your subscription usage by using the Subscriptions service on the {RHCloud}.
+
+You can configure host-based subscriptions for {EL} VMs on the following virtualization platforms:
+
+* VMware vSphere
+* {EL} Virtualization (KVM)
+* {OpenStack}
+* {oVirt}
```

Comment:
Downstream we don't support `{oVirt}` anymore and the project upstream has been abandoned. We remove support for it here https://github.com/theforeman/foreman_virt_who_configure/pull/136